IntegrationWithPharmacies/Controllers/SharingPrescriptionController.cs

```diff
@@ -1,30 +1,198 @@
-﻿using HealthClinic.CL.DbContextModel;
-using HealthClinic.CL.Service;
-using Microsoft.AspNetCore.Mvc;
-
-namespace IntegrationWithPharmacies.Controllers
-{
-
-    [Route("api/[controller]")]
-    [ApiController]
-    public class SharingPrescriptionController : Controller
-    {
-        private MedicineService MedicineService { get; set; }
-        public SharingPrescriptionController(MyDbContext context)
-        {
-            MedicineService = new MedicineService(context);
-        }
-        [HttpGet]
-        public IActionResult Get()
-        {
-            return Ok(MedicineService.GetAll());
-        }
-
-        [HttpPost]
-        public IActionResult Post(RequestedMedicine requested)
-        {
-            return Ok();
-        }
-
-    }
-}
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Net;
+using System.Net.Mail;
+using System.Text;
+using HealthClinic.CL.DbContextModel;
+using HealthClinic.CL.Service;
+using IntegrationWithPharmacies.FileProtocol;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging.Abstractions;
+using RestSharp;
+
+namespace IntegrationWithPharmacies.Controllers
+{
+
+    [Route("api/[controller]")]
+    [ApiController]
+    public class SharingPrescriptionController : Controller
+    {
+        private String Environment { get; set; }
+        private MedicineService MedicineService { get; set; }
+        private PatientService PatientService { get; set; }
+        public SharingPrescriptionController(MyDbContext context)
+        {
+            MedicineService = new MedicineService(context);
+            PatientService = new PatientService(context);
+            Environment = "Development"; 
+        }
+
+        [HttpGet("patients")]
+        public IActionResult GetPatients()
+        {
+            return Ok(PatientService.GetAll());
+        }
+        [HttpGet]
+        public IActionResult Get()
+        {
+            return Ok(MedicineService.GetAll());
+        }
+
+        [HttpGet("medicinesIsa")]
+        public IActionResult GetMedicinesFromIsa()
+        {
+            var client = new RestSharp.RestClient("http://localhost:8082");
+            var request = new RestRequest("/medicineRequested");
+            var response = client.Get<List<MedicineName>>(request);
+            Console.WriteLine("Status: " + response.StatusCode.ToString());
+            List<MedicineName> result = response.Data;
+            result.ForEach(medicine => Console.WriteLine(medicine.ToString()));
+            return Ok(result);

```

Comment:
Write this in form:
"response.Data.ForEach(medicine => Console.WriteLine(medicine.ToString()));
return Ok(response.Data);"